agent/api/statechange.go

```diff
@@ -15,6 +15,7 @@ package api
 
 import (
 	"fmt"
+	"github.com/aws/amazon-ecs-agent/agent/statechange"
```

Comment:
Usually we try to order imports like this:

```go
import (
	"standard/lib1"
	"standard/lib2"

	"github.com/third/party/package"
	"github.com/another/third/party/package"

	"github.com/aws/amazon-ecs-agent/agent/our/package"
)
```

`goimports` will maintain the sections for you (and handle sorting alphabetically) when you have a blank line between sections.